apps/server/src/tasks/SendTestUserEmail.ts

```diff
@@ -0,0 +1,31 @@
+import { fixedGroups } from '@raise/shared';
```

Comment:
nit: by convention and for consistency we generally name files with camelCase (the exception is React components with which name in PascalCase because of React's history where they used to be classes rather than functions)

future: optional: it'd be neat to embed this in the ESLint config (in general, any nit-like style changes that can be automated, should probably be automated - makes it less effort for people to comply, less effort for reviews, and generally lest angst all round)

[I also can't remember if I shared this before, if not [my review doc](https://adamjones.me/reviews) may be relevant]

Reply:
Thank you for the review doc! Will do my best to follow it as best as possible from now on!

Reply:
btw on the review doc, the main reason I shared it was to help clarify what I meant by nit/consider/future/optional etc. - I don't think you need to change what you're up to much based on that doc I think :)